### PR TITLE
WEBUI-1171: Forward-Port:Fix direct download URL failing with appended clientReaso…

### PR DESCRIPTION
**Pull request by swarnadipa-dev**
…n parameter: FORWARD PORT

### CODE REVIEW

**Review by Nishant0928 — DISMISSED** [review]

File: elements/nuxeo-document-blob/nuxeo-document-blob.js [hunk 1]
```diff
@@ -120,4 +120,11 @@ Polymer({
     }
     return obj;
   },
+
+  _getDocumentBlobUrl() {
+    if (this.blob) {
+      return this.blob && this.blob.downloadUrl ? this.blob.downloadUrl : this.blob.data;
```
Comment (rajatgoel-gl) [inline elements/nuxeo-document-blob/nuxeo-document-blob.js]: repetitive code. kindly remove multiple checks of this.blob

File: elements/nuxeo-document-blob/nuxeo-document-blob.js [hunk 2]
```diff
@@ -120,4 +120,11 @@ Polymer({
     }
     return obj;
   },
+
+  _getDocumentBlobUrl() {
```
Comment (rajatgoel-gl) [inline elements/nuxeo-document-blob/nuxeo-document-blob.js]: use same naming convention '_getBlobDownloadUrl'

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]

File: elements/nuxeo-document-blob/nuxeo-document-blob.js (hunk 2 above)
Comment (swarnadipa-dev) [reply elements/nuxeo-document-blob/nuxeo-document-blob.js]: done

File: elements/nuxeo-document-blob/nuxeo-document-blob.js (hunk 1 above)
Comment (swarnadipa-dev) [reply elements/nuxeo-document-blob/nuxeo-document-blob.js]: done

**Review by Nishant0928 — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**